### ISSUE TITLE
UDF are not working anymore after upgrade of Exasol 6.2.0 to 7.0.3

### ISSUE THREAD

**Issue by nicolamarangoni** [exasol/hadoop-etl-udfs#52]
Hello,

after upgrading Exasol from 6.2.0 to 7.0.3 the Hadoop-UDF aren't working anymore.
Here the error message:

```
SQL Error [22002]: VM error: F-UDF-CL-LIB-1127: F-UDF-CL-SL-JAVA-1002: F-UDF-CL-SL-JAVA-1013: 
com.exasol.ExaUDFException: F-UDF-CL-SL-JAVA-1080: Exception during run 
java.lang.ExceptionInInitializerError
org.apache.hadoop.util.ReflectionUtils.<clinit>(ReflectionUtils.java:137)
org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.initRegistry(MetricsSourceBuilder.java:92)
org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.<init>(MetricsSourceBuilder.java:56)
org.apache.hadoop.metrics2.lib.MetricsAnnotations.newSourceBuilder(MetricsAnnotations.java:42)
org.apache.hadoop.metrics2.impl.MetricsSystemImpl.register(MetricsSystemImpl.java:223)
org.apache.hadoop.metrics2.MetricsSystem.register(MetricsSystem.java:60)
org.apache.hadoop.security.UserGroupInformation$UgiMetrics.create(UserGroupInformation.java:120)
org.apache.hadoop.security.UserGroupInformation.<clinit>(UserGroupInformation.java:236)
com.exasol.hadoop.HdfsSerDeImportService.importFiles(HdfsSerDeImportService.java:88)
com.exasol.hadoop.scriptclasses.ImportHiveTableFiles.run(ImportHiveTableFiles.java:84)
com.exasol.ExaWrapper.run(ExaWrapper.java:196)
Caused by: java.lang.NullPointerException
java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2646)
java.base/java.lang.Runtime.loadLibrary0(Runtime.java:830)
java.base/java.lang.System.loadLibrary(System.java:1873)
java.management/java.lang.management.ManagementFactory.lambda$static$8(ManagementFactory.java:1020)
java.base/java.security.AccessController.doPrivileged(Native Method)
java.management/java.lang.management.ManagementFactory.<clinit>(ManagementFactory.java:1019)
	... 15 more
 (Session: 1684314033601904644)
```

Regards,

Nicola

**Comment by morazow** [comment]
Hello @nicolamarangoni,

Thanks for the feedback! 

I am going to check this on our site. Could you please tell me with which Hadoop distribution was the jar compiled?

**Comment by morazow** [comment]
From other issue you create, I see that you use `cdh5.16.2`.

**Comment by nicolamarangoni** [comment]
Hi @morazow I think we compiled it with the former version of CDH. It was 5.7.4.
However, as you can see from the other issue I posted, I cannot compile it because of an error in Maven.
I would like to compile a new jar and upload it, but issue 53 need to be solved first.

**Comment by nicolamarangoni** [comment]
This issue is becoming quit urgent because several people cannot work at the moment. Please make at list possible to compile this library with maven.

Best regards,
Nicola

**Comment by morazow** [comment]
Hello @nicolamarangoni,

We are looking into it. Meanwhile, my colleague prepared the steps to use the previous version of the language container. Please check the support ticket for the instructions.

**Comment by morazow** [comment]
Hey @nicolamarangoni,

Thanks for the feedback! 

We have finally fixed the issue in #58. Please let us know if there are any other issues.

Best,
Muhammet
